addons/l10n_it_edi_sdicoop/models/account_invoice.py

```diff
@@ -22,3 +22,10 @@ def send_pec_mail(self):
         # With SdiCoop web-service, no need to send PEC mail.
         # Set the state to 'other' because the invoice should not be managed par l10n_it_edi.
         self.l10n_it_send_state = 'other'
+
+    @api.depends('l10n_it_edi_transaction')
+    def _compute_show_reset_to_draft_button(self):
+        super(AccountMove, self)._compute_show_reset_to_draft_button()
+        self.filtered(lambda move: move.show_reset_to_draft_button and not move.l10n_it_edi_transaction).write({
```

Comment:
@lordkrandel A write in a compute is out of the question.

Reply:
Hummm, ok, then I just revert your and Antoine's suggestion?
https://github.com/odoo/odoo/pull/113085#discussion_r1112901375

Reply:
```py
    @api.depends('l10n_it_edi_transaction')
    def _compute_show_reset_to_draft_button(self):
        super(AccountMove, self)._compute_show_reset_to_draft_button()
        for move in self:
            move.show_reset_to_draft_button = move.show_reset_to_draft_button and not move.l10n_it_edi_transaction
```